### PR TITLE
Feature/elasticsearch box

### PR DESCRIPTION
**Pull request by LucasIME**

### CODE REVIEW

**Comment by villasv** [comment]
Huh, it shouldn't have failed. Apparently nose is finding tests inside flask that we shouldn't be running.

**Comment by villasv** [comment]
Nice. It also fixed the wrong coverage report that was considering the webapp as part of the package. I've also sneaked in another commit removing our in-house pylint service in favor of scrutinizer.

File: webapp/server.py [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+from elasticsearch import Elasticsearch
+import os
 
 app = Flask(__name__)
 
 @app.route('/')
 def home():
     return render_template('home.html')
 
+@app.route('/elastic')
```
Comment (villasv) [inline webapp/server.py]: Not a big deal, but I'd suggest this to be `/search` instead.

File: webapp/server.py [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+def elastic():
+    es = Elasticsearch(os.environ['BONSAI_URL'])
+    query = request.args['query']
+    response = es.search(index='minute', q=query)
+    return jsonify(response)
+
 if __name__ == '__main__':
     app.run()
```
Comment (villasv) [inline webapp/server.py]: Put the goddamn EOLs.

File: webapp/static/scripts/footer/footer.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<footer>Vigilante Político</footer>
```
Comment (villasv) [inline webapp/static/scripts/footer/footer.html]: Huh, this doesn't render into real HTML. Are you mixing React with Angular?

File: webapp/static/scripts/navbar/navbar.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-default navbar-static-top">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+                <img alt="Brand" src="static/img/favicon.ico">
```
Comment (villasv) [inline webapp/static/scripts/navbar/navbar.html]: Also not rendering here, no idea why. If I change the file to `icon.jpg` it works, so maybe there's something wrong with the favicon file?

File: webapp/server.py (hunk 1 above)
Comment (LucasIME) [inline webapp/server.py]: Changed it. What about /minutes? We're probably going to be searching for more stuff in the future...

File: webapp/server.py (hunk 2 above)
Comment (LucasIME) [inline webapp/server.py]: Fixed

**Comment by villasv** [comment]
Footer still not rendering correctly, and the logo is badly sized because I made it use the `jpg` file. I'll merge anyway, we'll fix it later.

**Pull request merged**